src/Debugger/bin/google-cloud-debugger

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Debugger;
+
+// Load the autoloader
+if (file_exists(__DIR__ . '/../../../vendor/autoload.php')) {
+    // Called from local git clone.
+    require_once __DIR__ . '/../../../vendor/autoload.php';
+} elseif (file_exists(__DIR__ . '/../../../autoload.php')) {
+    // Called from google/cloud-core installation.
+    require_once __DIR__ . '/../../../autoload.php';
+} elseif (file_exists(__DIR__ . '/../../../../../autoload.php')) {
+    // Called from google/cloud installation.
+    require_once __DIR__ . '/../../../../../autoload.php';
+} else {
+    die('No autoloader found');
+}
+
+function showUsageAndDie()
+{
+    $filename = basename(__FILE__);
+    fwrite(STDERR, <<<EOS
+Usage:
+  $filename [options] | <source-root>
+
+Options:
+  -c, --config=CONFIG_FILE       If specified, load this file which should initialize and return a Debugger\Daemon instance.
+  -s, --source-root=SOURCE_ROOT  Sets the source root for the Daemon. Ignored if --config is specified.
+
+
+EOS
+);
+    die();
+}
+
+if (count($argv) < 2) {
+    showUsageAndDie();
+}
+
+$options = getopt('c:s:', ['config:', 'source-root:'], $optind) + [
+    'c' => null,
+    'config' => null,
+    's' => null,
+    'source-root' => null
+];
+$config = $options['c'] ?: $options['config'];
+if ($config) {
+    if (file_exists($config)) {
+        $daemon = require_once $config;
+        var_dump($daemon);
```

Comment:
Do you want to keep `var_dump`?

Reply:
Not intentionally included. It'll be removed.